lunes_cms/api/v1/serializers/group_serializer.py

```diff
@@ -27,16 +27,20 @@ class Meta:
             "total_discipline_children",
         )
 
-    def get_total_discipline_children(self, obj):
+    def get_total_discipline_children(self, group):
         """Returns the total child count of a group.
         A child itself or one of its sub-children needs to
         contain at least one training set.
 
-        :param disc: Discipline instance
-        :type disc: ~lunes_cms.cms.models.discipline.Discipline
```

Comment:
I am pretty sure that this is a copy-and-paste error from the `discipline_serializer`